mesonbuild/interpreter/compiler.py

```diff
@@ -317,17 +319,23 @@ def symbols_have_underscore_prefix_method(self, args: T.List['TYPE_var'], kwargs
         return self.compiler.symbols_have_underscore_prefix(self.environment)
 
     @typed_pos_args('compiler.has_member', str, str)
-    @typed_kwargs('compiler.has_member', *_COMMON_KWS)
+    @typed_kwargs('compiler.has_member', *_HAS_KWS)
     def has_member_method(self, args: T.Tuple[str, str], kwargs: 'CommonKW') -> bool:
```

Comment:
```diff
- kwargs: 'CommonKW'
+ kwargs: 'HasKW'
```

?

Reply:
HasKW is not defined, you need to do the same as HeaderKW, or better just rename that one.